### ISSUE TITLE
Command line parsed twice

### ISSUE THREAD

**Issue by blattms** [OPM/opm-simulators#1836]
Just noticed that the command gets parsed twice. The first time at [flow.cpp#L148](/OPM/opm-simulators/blob/1a8453e52a09fe60dc8d479d9b1b1f48889dfffe/flow/flow.cpp#L148)

**Comment by andlaus** [comment]
no, it is not mandatory, but it does not do harm. do you care about the wasted 5 ms during each run?
